tensorflow/compiler/xla/service/gpu/tests/hlo_to_llvm_ir.cc

```diff
@@ -75,7 +75,7 @@ xla::Status CompileAndPrintLlvmIr(const std::string& hlo_text,
   std::string target_triple = "nvptx64-nvidia-cuda";
   std::string datalayout = "nvptx64-nvidia-cuda";
   std::string platform_name = "CUDA";
-#else
+#elif TENSORFLOW_USE_ROCM
```

Comment:
Actually, this part doesn't need #elif TENSORFLOW_USE_ROCM, and it will cause build failed. #else works here. .